mysite/urls.py

```diff
@@ -15,10 +15,18 @@
     2. Add a URL to urlpatterns:  path('blog/', include('blog.urls'))
 """
 
+from django.conf import settings
 from django.contrib import admin
 from django.urls import include, path
 
 urlpatterns = [
     path("", include("recipes.urls")),
     path("admin/", admin.site.urls),
 ]
+
+if settings.DEBUG:
```

Comment:
ah ok similar, just that I am not adding it to installed apps and middleware when not in debug

Reply:
not sure if that's best practice but it does not make sense to me